src/Silverback.Integration/Messaging/Inbound/ErrorHandling/MoveMessageErrorPolicy.cs

```diff
@@ -140,6 +140,7 @@ public override bool CanHandle(ConsumerPipelineContext context, Exception except
             private async Task PublishToNewEndpointAsync(IRawInboundEnvelope envelope, Exception exception)
             {
                 envelope.Headers.AddOrReplace(DefaultMessageHeaders.SourceEndpoint, envelope.ActualEndpointName);
```

Comment:
This header should be replaced by `x-source-topic` (for Kafka and MQTT at least).

Reply:
Done and moved to `KafkaMessageHeaders`.